lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -681,6 +683,26 @@ func (c *ContainerProxy) StreamContainerStats(ctx context.Context, config *conve
 	return nil
 }
 
+func (c *ContainerProxy) GetContainerChanges(vc *viccontainer.VicContainer) (io.ReadCloser, error) {
```

Comment:
I believe this entry point is constrained by the docker API logic- they could both call a common function however.